nltools/data/adjacency.py

```diff
@@ -383,32 +386,34 @@ def squareform(self):
                 return [x.data.reshape(int(np.sqrt(x.data.shape[0])),
                                        int(np.sqrt(x.data.shape[0]))) for x in self]
 
-    def plot(self, limit=3, *args, **kwargs):
+    def plot(self, limit=3, axes=None, *args, **kwargs):
```

Comment:
Do you want to add a quick doc string for the plot function? It looks like one of the methods that doesn't have one

Reply:
np. it's essentially just wrapper for heatmap.

Reply:
done.